engine/runtime-integration-tests/src/test/java/org/enso/compiler/test/IfThenElseTest.java

```diff
@@ -143,7 +129,7 @@ public void conditionMustBeBoolean() throws Exception {
   }
 
   @Test
-  public void javaScriptBooleanIsSupported() throws Exception {
+  public void javaScriptBooleanIsSupported() {
```

Comment:
Tell your IDE to not remove `throws Exception` please - e.g. not to perform unnecessary code changes.